src/source/PeerConnection/SessionDescription.c

```diff
@@ -198,6 +198,29 @@ STATUS setPayloadTypesFromOffer(PHashTable codecTable, PSessionDescription pSess
                 CHK_STATUS(STRTOUI64(attributeValue, end - 1, 10, &parsedPayloadType));
                 CHK_STATUS(hashTableUpsert(codecTable, RTC_CODEC_ALAW, parsedPayloadType));
             }
+
+            if ((end = STRSTR(attributeValue, RTX_VALUE)) != NULL) {
+                CHK_STATUS(STRTOUI64(attributeValue, end - 1, 10, &rtxPayloadType));
+                attributeValue = pMediaDescription->sdpAttributes[++currentAttribute].attributeValue;
+                if ((end = STRSTR(attributeValue, RTX_CODEC_VALUE)) != NULL) {
+                    CHK_STATUS(STRTOUI64(end + STRLEN(RTX_CODEC_VALUE), end + STRLEN(RTX_CODEC_VALUE)+ 10, 10, &parsedPayloadType));
```

Comment:
Spacing but most importantly, why are you hard-coding end + STRLEN(RTX_CODEC_VALUE) + 10? Where is this 10 coming from? there is no comments at all. Also, could you #define this?